src/components/Signup/ShowNewPod.jsx

```diff
@@ -5,33 +5,38 @@ import React from 'react';
 import Typography from '@mui/material/Typography';
 
 // Hooks imports
-import useSession from '@hooks/useSession';
 import { useContactsList } from '@hooks';
+import { Button } from '@mui/material';
```

Comment:
Very minor, recommend import style to be `import Button from '@mui/material/Button'` to be consistent with other MUI imports